syserol/figures/figure9.py

```diff
@@ -0,0 +1,140 @@
+"""
+This creates Figure 9, Paper Figure 2.
+"""
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+from syserol.regression import function_elastic_net, two_way_classifications
+from syserol.dataImport import importFunction, createCube
+from syserol.model import (
+    Function_Prediction_10FoldCV,
+    SVM_2class_predictions,
+)  
+from sklearn.metrics import r2_score
+from syserol.figures.common import subplotLabel, getSetup
+from syserol.tensor import perform_CMTF
+
+
+def makeFigure():
+    """ Show Similarity in Prediction of Alter Model and Our Model"""
+    # Gather Function Prediction Accuracies
+    arry = Function_Prediction_10FoldCV(6)  # Our Function Predictions in an array
+    _, mapped = importFunction()
+    accuracies = np.zeros(12)
+    for ii, func in enumerate(mapped):
+        _, _, acc = function_elastic_net(func)  # Alter Function Predictions
+        accuracies[ii] = acc  # store accuracies
+    for i in np.arange(6):
+        x = arry[0:181, i]
+        y = arry[0:181, i + 6]
+        idx = np.isfinite(x)
+        accuracies[i + 6] = np.sqrt(
+            r2_score(x[idx], y[idx])
+        )  # Calculate our accuracies & store
+
+    # Create DataFrame
+    model = np.array(
+        [
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+        ]
+    )
+    function = np.array(
+        [
+            "ADCD",
+            "ADCC",
+            "ADNP",
+            "CD107a",
+            "IFNy",
+            "MIP1b",
+            "ADCD",
+            "ADCC",
+            "ADNP",
+            "CD107a",
+            "IFNy",
+            "MIP1b",
+        ]
+    )
+    data = {"Accuracy": accuracies, "Model": model, "Function": function}
```

Comment:
I'm only seeing "Alter model" in (a).

Reply:
Alter model in (b) comes from two_way_classifications (line 73).